packages/server/src/adapters/aws-lambda/index.ts

```diff
@@ -110,32 +118,12 @@ export function awsLambdaRequestHandler<
       return await opts.createContext?.({ event, context, ...innerOpts });
     };
 
-    const [contentTypeHandler, unsupportedMediaTypeError] =
-      selectContentHandlerOrUnsupportedMediaType(
-        [getLambdaHTTPJSONContentTypeHandler<TRouter, TEvent>()],
-        {
-          ...opts,
-          event,
-          req,
-        },
-      );
-
-    const response = await resolveHTTPResponse({
+    const response = await resolveResponse({
```

Comment:
how do we reject unsupported media types?

Reply:
`getContentTypeHandlerOrThrow()`